webapp/src/main/java/uk/gov/dvsa/motr/web/serialisation/LocalDateDeserialiser.java

```
@@ -4,17 +4,19 @@
 import com.fasterxml.jackson.databind.DeserializationContext;
 import com.fasterxml.jackson.databind.JsonDeserializer;
 
-import java.io.IOException;
 import java.time.LocalDate;
 
 public class LocalDateDeserialiser extends JsonDeserializer<LocalDate> {
 
 
     @Override
-    public LocalDate deserialize(JsonParser jsonParser, DeserializationContext deserializationContext)
-            throws IOException {
+    public LocalDate deserialize(JsonParser jsonParser, DeserializationContext deserializationContext) {
 
-        return toDateFromString(jsonParser.getText());
+        try {
+            return toDateFromString(jsonParser.getText());
+        } catch (Exception e) {
+            return null;
```

Why suppressing of the exception here?  If the LocalDate can not be serialised correctly we should throw the exception so it can be logged as an error.